### DISCUSSION TITLE
Is there any way that files in the sandbox Explorer can be dragged out of the sandbox

### DISCUSSION THREAD

**Discussion by lmou523** [sandboxie-plus/Sandboxie#3001] [Various ideas and suggestions]
I find that programs in the sandbox can't be dragged out of the sandbox right now, but sometimes I need files in the sandbox.
Does it support sandbox Explorer files and does it support drag and drop out

**Comment by bastik-1001** [comment]
There are ways to get files out of the box, but those are different from what you are looking for.

The following assumes that you are using the plus version.

Right-click on a box > Box Content > Browse Files: gives you a tree view of the contents from where you can recover individual files, via a right-click menu.
Right-click on a box > Box Content > Explore Content: Opens the root folder of the sandbox in an unsandboxed explorer window, from where you can copy or move files.
Right-click on a box > Recover Files: Opens a dialog, where you can select to show all files and select the files you want to recover.

**Reply by lmou523** [reply]
> There are ways to get files out of the box, but those are different from what you are looking for.
> 
> The following assumes that you are using the plus version.
> 
> Right-click on a box > Box Content > Browse Files: gives you a tree view of the contents from where you can recover individual files, via a right-click menu. Right-click on a box > Box Content > Explore Content: Opens the root folder of the sandbox in an unsandboxed explorer window, from where you can copy or move files. Right-click on a box > Recover Files: Opens a dialog, where you can select to show all files and select the files you want to recover.

Thank you for your answer. But that's not what I want. I'm a developer. I'm studying the technical principle of why you can't drag from inside the sandbox to outside.

**Comment by lmou523** [comment]
Dear @DavidXanatos,Can you help me out?
Why can't you drag inside the sandbox and out of the sandbox? (Because of function hooks?)
As I understand it, this piece has to do with ole. So if the process has enough permissions in the sandbox, it should get the data.
Or does this action involve dcomlaunch, which results in data transmission failure?
